README.md

```diff
@@ -9,7 +9,8 @@
 | Java       |É uma linguagem que ganhou força nos últimos tempos, sendo capaz de rodar em "qualquer lugar", além de ser bastante utilizada para aprendizado por conta da Orientação à Objeto. | É necessário muito código para se fazer pouca coisa com a linguagem, além do que, apesar de ser utilizada para ensino, é uma linguagem muito pesada para iniciantes, como abordado na [issue#2](https://github.com/mathvbarone/awesome-programming-languages/issues/2). Outro ponto a ser tocado, é a performance da mesma. |
 | Javascript |  O Javascript é uma das linguagens mais importantes da atualidade. Tem evoluído absurdamente nos ultimos anos, e o profissional que tem conhecimento nessa linguagem é bastante requisitado no mercado. Por ter uma sintaxe simples, em poucos segundos já é possível ver um "Hello World" na tela. É obrigatório o aprendizado dela para quem pretende trabalhar com desenvolvimento web.    |         |
 | Kotlin      |      |         |
-| PHP        |      |         |
+| PHP|É uma linguagem de script utilizada para a web, tem como foco ser Sever-Side, ou seja, atua no lado do servidor. É totalmente interpretada, não precisa ser compilada antes de ser executada.|Possui alguns problemas com compatibilidade entre as versões, e, é necessário por parte do programador tomar alguns cuidados com falhas de segurança para evitar ataques e invasões.|
```

Comment:
Cara, vou ser meio ~chato~ crítico aqui. Tanto o Pró quanto Contra que estão aí servem pra qualquer linguagem web backend. Não consegui enxergar nada ali como um pró ou contra da linguagem em si. Inclusive, não consigo achar que linguagem interpretada é um pró. Na minha opinião depende do cenário pra saber se é bom ou ruim. Enfim, ficou extremamente genérico tudo.

Qual a vantagem em programar em PHP e não Python ou Ruby? Qual a desvantagem? Acho que deveria achar algo que seja característica da linguagem específica, o que também pode entrar envolvimento em comunidade, documentação fácil, curva de aprendizado, quantidade de ferramentas...

Reply:
Por isso que é importante esperar um pouco pra dar merge ahahaha.

Bons pontos levantados @rafaels88

Reply:
@rafaels88  Nunca programei em Python ou Ruby, já não saberia dizer as diferenças entre eles. haha . Na minha opinião, bem limitada por sinal, foi o que considerei como vantagens e desvantagens em relação as linguagens que conheço, como C++, Java. Mas, contribua e fique a vontade para alterar tudo o que for necessário. A ideia é essa.

Reply:
@rafaels88 vamos fazer assim...vou aprovar esse PR do @wilsonjuniorwj, e você abre um novo PR com as alterações que acham necessárias, pode ser?

Reply:
Não me sinto muito confortável sem antes ter isso aqui https://github.com/mathvbarone/awesome-programming-languages/issues/13 Mas manda ver =)